### DISCUSSION TITLE
Files are out of order I can't find in libarary place search every index files [rubydevkit3.3.5x64]is install but try add I have to delete Ruby in order to reinstall no space  Windows(x86) pardon I was out of line but Microsoft one employe a cursing at me bad words! Check at pull request dept

### DISCUSSION THREAD

**Discussion by Rockyspade** [community/community#146095] [Enterprise] [answered]
### Select Topic Area

Question

### Body

This enterprise I have no axisPC Microsoft purpose block my account and even GitHub I can't make an files in dependence repos not letting do my work in Until Intel stop them and check my everything is all ok passed the PC all the  Windows(x86)Microsoft trying keep my repos in PC @queenofcorgis

**Comment by queenofcorgis** [comment, marked as answer]
Hi there,

If Microsoft is the origin of the situation you are facing, you will have to contact [their Support team](https://support.microsoft.com/en-us/contactus).

The only place to go for answers to for GitHub account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

Any repeat and future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
